src/platform/plugins/shared/esql/public/triggers/esql_controls/control_flyout/identifier_control_form.tsx

```diff
@@ -98,35 +117,48 @@ export function FieldControlForm({
   const isControlInEditMode = useMemo(() => !!initialState, [initialState]);
 
   useEffect(() => {
```

Comment:
it's easier to follow what effect does using a named function 
```suggestion
  useEffect(function initAvailableIdentifiersOptions() {
```